Controller/DynamicController.php

```diff
@@ -30,7 +29,7 @@
 /**
  * Controller to create dynamic form entries list.
  */
-class DynamicController implements ClassResourceInterface
+class DynamicController
```

Comment:
I don't think we removed in any bundles yet the `ClassResourceInterface` on 2.x so we should keep it here.